content/en/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -62,7 +62,7 @@ That's how Kubernetes comes to the rescue! Kubernetes provides you with a framew
 Kubernetes provides you with:
 
 * **Service discovery and load balancing**
-Kubernetes can expose a container using the DNS name or using their own IP address. If traffic to a container is high, Kubernetes is able to load balance and distribute the network traffic so that the deployment is stable.
```

Comment:
My two cents: I find the original text somewhat easier to understand, but also like some of the new content.
Kubernetes exposes a container using the DNS [domain] name or the container's IP address. If network traffic to a container is high, Kubernetes can distribute and load balance the traffic across containers.

Reply:
So What do you think should the final output be focused upon?